centre-de-recherche-sur-les-civilisations-de-l-asie-orientale.csl

```diff
@@ -5,7 +5,6 @@
     <title-short>CRCAO</title-short>
     <id>http://www.zotero.org/styles/centre-de-recherche-sur-les-civilisations-de-l-asie-orientale</id>
     <link href="http://www.zotero.org/styles/centre-de-recherche-sur-les-civilisations-de-l-asie-orientale" rel="self"/>
-    <link href="https://github.com/citation-style-language/styles/pull/1385#issuecomment-73234582" rel="documentation"/>
```

Comment:
Is there a reason for deleting this "documentation" link? We try to have a link for all our styles that points to the style guide. In this case, the GitHub comment the style linked to says "they haven't a documentation for the style", which is good for us to know.